tests/test_downloads.py

```
@@ -0,0 +1,139 @@
+
+from httpie.compat import urlopen
+from httpie.downloads import (
+    parse_content_range, filename_from_content_disposition, filename_from_url,
+    get_unique_filename, ContentRangeError, Download,
+)
+from utils import http, TestEnvironment
+
+
+class Response(object):
+    # noinspection PyDefaultArgument
+    def __init__(self, url, headers={}, status_code=200):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)